sample-kotlin-model/src/main/kotlin/com/vimeo/dummy/sample_kotlin/BoolFields.kt

```diff
@@ -9,12 +9,7 @@ import com.vimeo.stag.UseStag
  * Created by anthonycr on 9/2/17.
  */
 @UseStag
-class BoolFields {
-
-    @SerializedName("test1")
-    var test1: Boolean? = null
-
-    @SerializedName("test2")
-    var isTest2: Boolean? = null
-
-}
\ No newline at end of file
+data class BoolFields(
+        @SerializedName("test1") var test1: Boolean? = null,
```

Comment:
Why does this field require a `SerializedName` annotation with the same name as the variable?

Reply:
It does not require the annotation, it is just added for consistency.